mortgageAPI.yaml

```diff
@@ -2661,6 +2661,11 @@ components:
               type: array
               items:
                 $ref: '#/components/schemas/Asset'
+            residentialSituation:
+              type: string
+              enum:
+                - home_owner
+                - renter
```

Comment:
Please add an example to enable nice example views in the Swagger UI

Reply:
done